### ISSUE TITLE
 logistic

### ISSUE THREAD

**Issue by khawajaJunaid** [unifyai/ivy#22781]

**Comment by khawajaJunaid** [comment]
@AnnaTz this issue was opened by me and assigned to me, according to the contribution guide I was to work on this issue

**Comment by AnnaTz** [comment]
> @AnnaTz this issue was opened by me and assigned to me, according to the contribution guide I was to work on this issue

Hi @khawajaJunaid, you're right. I didn't notice the issue was opened by someone other than the PR author. Apologies for the oversight 😕

**Comment by AnnaTz** [comment]
@rsaeta for future contributions please refrain from opening PRs for issues that have been opened by other people. You are supposed to select one of the free functions from a ToDo list and open an issue for that yourself. Thanks!

**Comment by rsaeta** [comment]
Yes sorry, I missed this part in the contribution guide and thought to take any open issue. I have not merged this yet, and instead opened an issue/PR of my own. Again, sorry about taking the issue.

**Comment by khawajaJunaid** [comment]
It is unfortunate but I guess its alright @AnnaTz can I still work on this issue or will this be assigned to @rsaeta, would appreciate the clarity so I can either start work on it or look for another issue ?

**Comment by AnnaTz** [comment]
> It is unfortunate but I guess its alright @AnnaTz can I still work on this issue or will this be assigned to @rsaeta, would appreciate the clarity so I can either start work on it or look for another issue ?

I merged @rsaeta's PR already, 3 days ago, so the issue has been completed. Feel free to look for another issue @khawajaJunaid.
